File: Java/1-50/8.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        long a, b, c, d, e, mn, mx;

        a = scanner.nextLong();
        b = scanner.nextLong();
        c = scanner.nextLong();
        d = scanner.nextLong();
        e = scanner.nextLong();

        mn = Math.min(Math.min(a, b), Math.min(c, Math.min(d, e)));
        mx = Math.max(Math.max(a, b), Math.max(c, Math.max(d, e)));

        System.out.println((a + b + c + d + e - mx) + " " + (a + b + c + d + e - mn));
    }
}

File: Kotlin/1-50/8.kt
import java.util.*

fun main() {
    val scanner = Scanner(System.`in`)
    var a: Long
    var b: Long
    var c: Long
    var d: Long
    var e: Long
    var mn: Long
    var mx: Long

    a = scanner.nextLong()
    b = scanner.nextLong()
    c = scanner.nextLong()
    d = scanner.nextLong()
    e = scanner.nextLong()

    mn = minOf(minOf(a, b), minOf(c, minOf(d, e)))
    mx = maxOf(maxOf(a, b), maxOf(c, maxOf(d, e)))

    println("${a + b + c + d + e - mx} ${a + b + c + d + e - mn}")
}
